### PR TITLE
Added crop box to image pair distance function

### PR DESCRIPTION
**Pull request by tomasmahlberg**

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Pull requests to kean? Nice, it's been a long time.

File: source/draw/RasterMonochrome.ooc [hunk 1]
```diff
@@ -212,7 +217,7 @@ RasterMonochrome: class extends RasterPacked {
 						result += distance sqrt()
 					}
 				}
-			result /= this size norm
+			result /= hasCropBox ? cropBox area : this size norm
```
Comment (marcusnaslund) [inline source/draw/RasterMonochrome.ooc]: size norm != area

**Comment by marcusnaslund** [comment]
I'm OK to merge this though if ready

File: source/draw/RasterMonochrome.ooc [hunk 2]
```diff
@@ -182,14 +182,19 @@ RasterMonochrome: class extends RasterPacked {
 			result = this distance(converted)
 			converted referenceCount decrease()
 		} else {
+			hasCropBox := !cropBox hasZeroArea
+			startX := hasCropBox ? cropBox left : 0
+			startY := hasCropBox ? cropBox top : 0
+			endX := hasCropBox ? cropBox right : this size x
+			endY := hasCropBox ? cropBox bottom : this size y
 			sizeX := this size x
 			sizeY := this size y
```
Comment (marcusnaslund) [inline source/draw/RasterMonochrome.ooc]: But these can be removed now?

Comment (tomasmahlberg) [inline source/draw/RasterMonochrome.ooc]: Well, they're still used, but maybe there's not much gain in putting them in local variables now?

Comment (marcusnaslund) [inline source/draw/RasterMonochrome.ooc]: oh sorry, nvm then

**Comment by marcusnaslund** [comment]
_breaking the law_ :notes:

**Comment by marcusnaslund** [comment]
PRInt fails because they've added some experimental step we can ignore for now.

**Pull request merged**